packages/client/src/components/GrantsTable.vue

```diff
@@ -143,7 +143,7 @@ export default {
       searchInput: null,
       debouncedSearchInput: null,
       reviewStatusFilters: [],
-      opportunityStatusFilters: [],
+      opportunityStatusFilters: ['Forecasted', 'Posted'],
```

Comment:
Since we're going to be dropping forecasts for the time-being, shouldn't that be removed from the defaults on the new table? 
```suggestion
      opportunityStatusFilters: ['Posted'],
```

Reply:
This is the old search experience. The only change I wanted to do here is to make it default ignore the `Closed / Archived` grants for performance improvements.

The idea here was to enable deployment to production so we can get grants-ingest without breaking the old table.

Reply:
To confirm- the new experience, doesn't have forecasted grants at all. It is fully hidden from view.